packages/office-ui-fabric-react/src/components/pickers/PeoplePicker/PeoplePickerItems/PickerItemsDefault.scss

```diff
@@ -28,5 +28,11 @@
     vertical-align: middle;
   }
 
-
+  // People Picker specific override to prevent truncation of Persona name
+  .ms-Persona.ms-Persona--xs {
+    .ms-Persona-details {
+      flex-direction: row;
```

Comment:
Does this need flex-direction if it's not flex? Also, row is already the default value for flex-direction so it's unnecessary to specify it.